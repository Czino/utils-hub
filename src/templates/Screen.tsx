export const Screen = ({ children }: React.ComponentProps<'div'>) => <div className="p-4">{children}</div>
