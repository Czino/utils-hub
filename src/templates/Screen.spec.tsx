import { render } from '@testing-library/react'
import { Screen } from './Screen'

describe('Screen', () => {
  it('should render Screen', () => {
    const { asFragment } = render(<Screen />)
    expect(asFragment()).toMatchSnapshot()
  })
})
