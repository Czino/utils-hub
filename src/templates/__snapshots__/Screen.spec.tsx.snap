// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen should render Screen 1`] = `
<DocumentFragment>
  <div
    class="p-4"
  />
</DocumentFragment>
`;
